helm/csi-charts/templates/daemonset.yaml

```diff
@@ -38,6 +38,17 @@ spec:
             - -bind=unix://{{ .Values.kubeletPath }}/plugins/csi-exos-x.seagate.com/csi.sock
             - -chroot=/host
 {{- include "csidriver.extraArgs" .Values.node | indent 10 }}
+          env:
+            - name: EXOS-X-CSI-NODE-NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: spec.nodeName
+            - name: EXOS-X-CSI-NODE-IP
```

Comment:
Can we leave the branding out of the env var names?  It'll be problematic when we need to change the branding.

Reply:
Removed branding